packages/related-content/types.ts

```diff
@@ -0,0 +1,13 @@
+import type { CardProps } from '@hashicorp/react-card/types'
+import type { ThemeVariant } from '@hashicorp/react-button/types'
+export interface RelatedContentProps {
+  appearance?: 'light' | 'dark'
+  headline: string
+  description?: string
+  cards: Array<CardProps>
```

Comment:
💭 Possibly adjust CardProps to exclude appearance usage here for clarity.

Reply:
Updated [here](https://github.com/hashicorp/react-components/pull/731/commits/d43f0bff7a331468ebce57d688fd5b13a2dfd4e0).